client/store/modules/repository/actions.js

```diff
@@ -7,7 +7,7 @@ export const initialize = ({ commit, dispatch }, id) => {
   const getRoute = entity => `repositories/${id}/${entity}`;
   const modules = {
     activities: 'activities',
-    tes: 'content-elements',
+    elements: 'content-elements',
```

Comment:
We started to use content elements terminology because elements seemed a bit too generic (So I am not sure what to think about this). `contentElements` in some context seem a bit much and `elements` in some other seem a bit too generic.

Reply:
It is probably best to be more verbose. It is easier for people to know what it refers to, especially people less familiar with the project.

Reply:
I made the change inside the store so all the `getters` and `actions` reference `contentElements` but I did not make the changes throughout the app. I was not sure if that would be too excessive or not.